docs/architecture/adr-112-proposer-based-timestamps.md

```diff
@@ -292,17 +305,31 @@ The only change we will make to the prevote step is to what a validator consider
 The precommit step will not require much modification.
 Its proposal validation rules will change in the same ways that validation will change in the prevote step with the exception of the `timely` check: precommit validation will never check that the timestamp is `timely`.
 
+<!---
 ### Remove voteTime Completely
 
-[voteTime](https://github.com/cometbft/cometbft/blob/822893615564cb20b002dd5cf3b42b8d364cb7d9/internal/consensus/state.go#L2229) is a mechanism for calculating the next `BFTTime` given both the validator's current known Unix time and the previous block timestamp.
+[voteTime](https://github.com/cometbft/cometbft/blob/822893615564cb20b002dd5cf3b42b8d364cb7d9/internal/consensus/state.go#L2229) is a mechanism for calculating the next `BFT Time` given both the validator's current known Unix time and the previous block timestamp.
 If the previous block timestamp is greater than the validator's current known Unix time, then voteTime returns a value one millisecond greater than the previous block timestamp.
-This logic is used in multiple places and is no longer needed for proposer-based timestamps.
+This logic is used in multiple places and is no longer needed for PBTS.
 It should therefore be removed completely.
+--->
+
+### Backwards compatibility
+
+In order to ensure backwards compatibility, PBTS should be enabled using a consensus parameter.
+The proposed approach is similar to the one adopted to enable vote extensions:
+[`ABCIParams.VoteExtensionsEnableHeight`](https://github.com/cometbft/cometbft/blob/main/spec/abci/abci++_app_requirements.md#abciparamsvoteextensionsenableheight).
+
+In summary, the network will migrate from the `BFT Time` method for assigning
+and validating timestamps to the new method for assigning and validating
+timestamps adopted by `PBTS` from a given, configurable height.
```

Comment:
I would add here:
* Once PBTS is activated, there are no provisions for a chain to revert back to BFT Time (the latter being considered obsolete).
* We decided that blocksync will not run BFT Time validation in blocks created before PBTS was activated on the chain. The reasoning is that, if the light client algorithm accepts an already produced block, we assume the BFT Time algorithm was followed. This reasoning is analogous to why we don't check `timely` when precommiting a proposal with a polka.

Reply:
The second point is something worth discussing.

Reply:
We would need a section for that, if we are making this change (which I think it is not needed), see this [comment](https://github.com/cometbft/cometbft/pull/2205#discussion_r1480207485).

Reply:
Added the first point to this description.

Reply:
Thanks, and the second, we have anyway changed the decision, to not change the checks when in BFT Time